mod/isic/lib/formula/isic.rb

```diff
@@ -0,0 +1,13 @@
+module Formula
+  # Calculator class for ISIC division and group codes
+  class Isic < HardCoded
```

Comment:
Calculator no longer receives a card object. I created a new calculator classes for your hard coded cases to make them work again.
(and also fixed the bug that validation wasn't called)